### ISSUE TITLE
AccessViolationException When Changing Streams

### ISSUE THREAD

**Issue by thedude61636** [jacobbo/WebEye#76]
Using the included demo in this repo selecting a local stream then stopping and selecting another stream causes a System.AccessViolationException in BitmapExtentions.UpdateWith method 
![image](https://user-images.githubusercontent.com/13031288/74647610-05564580-518d-11ea-9ad6-190e125d4e98.png)

**Comment by jacobbo** [comment]
Is it always reproducible? Another local stream?

**Comment by thedude61636** [comment]
Yes it always happens, first starting a local stream stopping then starting another one 
This happens if I switch from stream 0 to 1 or from 1 to 0 both will cause the same error
I think one solution to this problem is clearing the bitmap before starting a stream

**Comment by ckrc** [comment]
Clearing the currentFrame and writeableBitmap did solve the problem
